src/RequestHandler.cpp

```diff
@@ -76,6 +78,25 @@ void RequestHandler::subscribeConsumeAtOffset(std::string TopicName,
   }
 }
 
+/// Ensures there are messages to read at offset provided by the user, otherwise
+/// throws an ArgumentsException.
+///
+/// \param Offset
+/// \param TopicName
+void RequestHandler::verifyOffset(int32_t Offset, std::string TopicName) {
```

Comment:
Both arguments can be const-ref here eg: 
`const int32_t Offset, const std::string TopicName`

this looks like it can also be applied to `verifyNLast` as well